app/scripts/main/controllers/url-selector.controller.js

```diff
@@ -0,0 +1,107 @@
+/*jshint -W003, -W098, -W033 */
+(function () {
+  'use strict';
+
+  /**
+   * @ngdoc function
+   * @name ngAmrsApp.controller:MainCtrl
+   * @description
+   * # MainCtrl
+   * Controller of the ngAmrsApp
+   */
+  angular
+    .module('ngAmrsApp')
+    .controller('UrlSelectorCtrl', UrlSelectorCtrl);
+  UrlSelectorCtrl.$nject = ['$rootScope', '$scope', 'OpenmrsSettings', 'EtlRestServicesSettings', 'AuthService', '$state'];
+
+  function UrlSelectorCtrl($rootScope, $scope, OpenmrsSettings, EtlRestServicesSettings, AuthService, $state) {
+
+    $scope.currentMode = '';
+
+    $scope.currentRestBaseUrl = '';
```

Comment:
Renaming currentRestBaseUrl to currentOpenmrsRestBaseUrl would be awesome to avoid confusion.